examples/nlp/language_modeling/megatron_gpt_prompt_tuning.sh

```diff
@@ -0,0 +1,28 @@
+#/bin/bash
+
+GPUS=2
+MAX_STEPS=3000
+PROMPT_LENGTH=10
+RESTORE_PATH='megatron_gpt.nemo'
+
+echo "Prompt tuning starting"
+python megatron_gpt_prompt_tuning.py \
+	--config-name=megatron_gpt_config \
+	trainer.gpus=$GPUS \
+	trainer.max_steps=$MAX_STEPS \
+	restore_from_path=$RESTORE_PATH \
+	+model.use_soft_prompts=True \
+	+model.prompt_length=$PROMPT_LENGTH \
+	+model.new_prompt_tags=['NER-Yes-No, NER-Complete'] \
```

Comment:
how does one specify number of tokens per tag?

Reply:
Each prompt tag has to have the same number of tokens during a given prompt tuning run. I would have to unbatch the data in the forward pass to accommodate different length soft prompts. So you just give one prompt length in the config file